src/checker.rs

```diff
@@ -23,7 +23,7 @@ impl Checker for ExecutableChecker {
             .and_then(|c| Ok(unsafe { libc::access(c.as_ptr(), libc::X_OK) == 0 }))
             .unwrap_or(false)
     }
-
+    
```

Comment:
unexpected spaces.

Reply:
are you saying you don't want these spaces or i should remove the line entirely?